mentorship ios/Service/Networking/LoginAPI.swift

```diff
@@ -23,6 +23,8 @@ class LoginAPI: LoginService {
             .sink { response in
                 var loginResponseData = LoginModel.LoginResponseData(message: response.message)
                 // if login successful, store access token in keychain
+                print("here testting the decdoed respose\(response)")
+//                CoreUserProfile().saveCore(token: response.accessToken!)
```

Comment:
remove this

Reply:
thanks for review, 
I will do the changes,
 I have used 2 loadCoredata() function so in future it is easy to customise for different data tables.
can you please say how will I ignore pods, the build is crashing without installing them